lib/hex_web/store.ex

```diff
@@ -33,24 +31,24 @@ defmodule HexWeb.Store do
   @callback delete_docs_page(key) :: term
   @callback send_docs_page(Plug.Conn.t, key) :: Plug.Conn.t
 
-  defdispatch list_logs(region, bucket, prefix),   to: impl
-  defdispatch get_logs(region, bucket, key),       to: impl
-  defdispatch put_logs(region, bucket, key, body), to: impl
-  defdispatch put_registry(body, signature),       to: impl
-  defdispatch put_registry_signature(body),        to: impl
-  defdispatch send_registry(conn),                 to: impl
-  defdispatch send_registry_signature(conn),       to: impl
-  defdispatch put_release(package, version, body), to: impl
-  defdispatch delete_release(key),                 to: impl
-  defdispatch send_release(conn, key),             to: impl
-  defdispatch put_docs(key, body),                 to: impl
-  defdispatch delete_docs(key),                    to: impl
-  defdispatch send_docs(conn, key),                to: impl
-  defdispatch put_docs_file(key, body),            to: impl
-  defdispatch put_docs_page(key, cdn_key, body),   to: impl
-  defdispatch list_docs_pages(prefix),             to: impl
-  defdispatch delete_docs_page(key),               to: impl
-  defdispatch send_docs_page(conn, key),           to: impl
-
-  defp impl, do: Application.get_env(:hex_web, :store_impl)
+  @store_impl Application.get_env(:hex_web, :store_impl)
```

Comment:
That might have been the reason for the custom `defdispatch` macro but since we don't seem to change the config during runtime I guess it's fine to make the change.